CMakeLists.txt

```diff
@@ -327,6 +327,7 @@ endif()
 include(GenerateVersionInformation)
 include(${CMAKE_CURRENT_LIST_DIR}/cmake/DetectGitRevision.cmake)
 configure_file("${PROJECT_SOURCE_DIR}/src/precice/impl/versions.hpp.in" "${PROJECT_BINARY_DIR}/src/precice/impl/versions.hpp" @ONLY)
+configure_file("${PROJECT_SOURCE_DIR}/src/precice/Version.h.in" "${PROJECT_BINARY_DIR}/src/precice/Version.h" @ONLY)
```

Comment:
Why do we have `Version.h` starting with a capital `V` and the `versions.hpp` starting with a small `v`?

Reply:
This is merely to stay consistent with the `SolverInterface.hpp` file. I used the same reasoning for the `Tooling.hpp` file.

Our naming mixture is strange though.